wagtail_newsletter/campaign_backends/mailchimp.py

```diff
@@ -45,9 +46,29 @@ def url(self) -> str:
         else:
             return f"{base_url}/campaigns/edit?id={self.web_id}"
 
+    def get_report(self) -> "dict[str, Any]":
```

Comment:
The type here is intentionally vague, so that it's easy to override this method, and the `{% block campaign_status %}` block, to show more (or different) information in the report.